.gitignore

```diff
@@ -1 +1,21 @@
-node_modules
+# See https://help.github.com/ignore-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
```

Comment:
Files related to your machine/os can be set globally in your global .gitignore
https://gist.github.com/subfuzion/db7f57fff2fb6998a16c